src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -5798,30 +5803,27 @@ and TcExprLazy (cenv: cenv) overallTy env tpenv (synInnerExpr, m) =
     let expr = mkLazyDelayed g m innerTy (mkUnitDelayLambda g m innerExpr)
     expr, tpenv
 
+and CheckTupleIsCorrectLength g (env: TcEnv) m tupleTy args typeCheckArgs =
```

Comment:
I guess we would use name `tcArgs` for the parameter - lower case version of the things being abstracted